packages/backend/src/open_payments/payment/incoming/service.test.ts

```diff
@@ -81,14 +83,17 @@ describe('Incoming Payment Service', (): void => {
           type: IncomingPaymentEventType.IncomingPaymentCreated
         })
       ).resolves.toHaveLength(0)
+      options.client = client
```

Comment:
we can remove this line since we are already adding client to the args. Or we remove L89 since we're spreading `options` in there anyway